static/js/biomedical/disease/chart.ts

```diff
@@ -247,3 +259,82 @@ export function drawDiseaseSymptomAssociationChart(
       (d) => `Symptom: ${d.name}<br>Odds Ratio Association: ${d.oddsRatio}`
     );
 }
+/**
+ * Draws the disease ontology hierarchy chart for the disease of interest
+ * @param id the div id where the chart is rendered on the page
+ * @param data the disease data passed into the function
+ */
+export function drawDiseaseOntologyHierarchy(
+  id: string,
+  data: DiseaseParentTree[]
+): void {
+  const height = GRAPH_HEIGHT - MARGIN.top - MARGIN.bottom;
+  const width = GRAPH_WIDTH - MARGIN.left;
+  // tree root
+  const root = d3.hierarchy(data).sort((a, b) => b.height - a.height);
+  // tree layout
+  const treeLayout = d3.tree().size([600, 120]);
```

Comment:
where are the numbers 600 and 120 coming form? should these be height and width?

Reply:
Yes, these should be width and height! Changed it!